src/ducks/apps/index.js

```diff
@@ -238,10 +238,11 @@ export function uninstallApp (slug) {
   }
 }
 
-export function installApp (slug, source) {
+export function installApp (slug, source, isUpdate = false) {
   return (dispatch, getState) => {
     dispatch({type: INSTALL_APP})
-    return cozy.client.fetchJSON('POST', `/apps/${slug}?Source=${source}`)
+    const verb = isUpdate ? 'PUT' : 'POST'
+    return cozy.client.fetchJSON(verb, `/apps/${slug}?Source=${encodeURIComponent(source)}`)
```

Comment:
I think this part should at least be encapsulated into an `app.js` library, it not in CozyClient. To be also usable without redux. What do you think ?

Reply:
I see your point, I think I'll have to do it in a second time but not in this PR.